### PR TITLE
Fixing the live tests for PPE endpoint

### PR DESCRIPTION
**Pull request by mssfang**
Since we are using the PPE endpoint for GA version, the new feature only exist in this resource endpoint. So we are the resource's api key and endpoint for temperately use. 

Created an issue for reverting it back after service has done the rolling service. #11052

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java [hunk 1]
```diff
@@ -52,6 +52,7 @@
     static final String INVALID_URL = "htttttttps://localhost:8080";
     static final String VALID_HTTPS_LOCALHOST = "https://localhost:8080";
     static final String FAKE_API_KEY = "1234567890";
+    static final String AZURE_TEXT_ANALYTICS_API_KEY = "AZURE_TEXT_ANALYTICS_API_KEY";
```
[the remark this answers is not in the document]
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java:55]: I can keep it here and remove AZURE_TEXT_ANALYTICS_API_KEY from `TextAnalyticsClientBuilderTest`

[the remark this answers is not in the document]
Comment (samvaity) [reply sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java:55]: But it's used in the Builder. Also, I see that is it used only in one place, do we need it as a constant then?

[the remark this answers is not in the document]
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java:55]: offline discussion. Keep it here in TextUtil.

**Pull request merged**